### PR TITLE
New topic about how to Add GTM (Google Tag Manager) For Magento 2 Stores

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds new topic about how to Add GTM (Google Tag Manager) For Magento 2 Stores

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- Adobe Commerce 

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/marketing/google-tag-manager-create-tag.html

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- https://developers.google.com/tag-manager/devguide

whatsnew
Updated the [Google Tag Manager](https://docs.magento.com/user-guide/marketing/google-tag-manager.html) to include the procedure for adding the GTM code snippets to the Commerce store.

### CODE REVIEW

**Comment by leslietilling** [comment]
@EvheniiaKolkhiieva  Seems like this should be a "Step 4" for this topic rather than a new topic?: https://docs.magento.com/user-guide/marketing/google-tag-manager.html  

We don't want to create many small topics. We will be removing the Troubleshooting topic in that section.

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , I have added "Step 4" for this topic -  https://docs.magento.com/user-guide/marketing/google-tag-manager.html, and delete previous changes. I can also remove the Troubleshooting topic in that section as part of this PR, what do you think?

File: src/marketing/google-tag-manager.md [hunk 1]
```diff
@@ -123,6 +123,29 @@ For additional information, see Google's [Container export and import][5]. These
...
+
+    ![Install Google Tag Manager]({% link images/images-ee/head-tag.png %}){: .zoom}
+    _HTML Tag inserting_
+
+1. Expand the **Footer** tab and paste the GTM code for <body> under the **Miscellaneous HTML**.
+
+    ![Install Google Tag Manager]({% link images/images-ee/footer-tag-section.png %}){: .zoom}
+    _Footer Tag inserting_
+
+1. When complete, click <span class="btn">Save Configuratio</span>.
```
Comment (leslietilling) [inline src/marketing/google-tag-manager.md:147]:
```suggestion
## Step 4. Add the GTM code to your Adobe Commerce store

1. Copy the GTM code:

   There are two GTM code snippets to be added to your Commerce site: the first one for the <head> tag and the and the second for the <body> tag.

    ![Install Google Tag Manager]({% link images/images-ee/install-google-tag-manager.png %}){: .zoom}
    _Google Tag Manager code_

1. In the Commerce Admin, go to **Content** > _Design_ > **Configuration** and open the store view in edit mode.

1. Under _Other Settings_, expand **HTML Head** and paste the code you copied from GTM for the <head> tag in the **Scripts and Style Sheets** field.

    ![Install Google Tag Manager]({% link images/images-ee/head-tag.png %}){: .zoom}
    _Inserting code in the HTML Head_

1. Expand **Footer** and paste the GTM code for <body> in the **Miscellaneous HTML** field.

    ![Install Google Tag Manager]({% link images/images-ee/footer-tag-section.png %}){: .zoom}
    _Inserting code in the Footer_

1. When complete, click <span class="btn">Save Configuration</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style, clarity, and branding

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you!
I'm done, please check and approve this PR 😊☀️

File: src/marketing/google-tag-manager.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/google-tag-manager.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**